dom-events.tar/dom-events/buttons.js

```diff
@@ -0,0 +1,27 @@
+function handleClick(event){
+  console.log('button clicked');
+  console.log(event);
+  console.log(event.target);
+}
+const element = document.querySelector('.click-button');
+  if (!$clickButton) throw new Error('The $clickBtn query failed');
+  element.addEventListener('click', handleClick);
+
+
+function handleMouseOver(event){
+  console.log('button hovered');
+  console.log(event);
+  console.log(event.target);
+}
+ const element2 =document.querySelector('.hover-button');
+  if (!$hoverButton) throw new Error('The $hoverButton query failed');
+  element2.addEventListener('mouseover',event)
```

Comment:
The second argument here should be `handleMouseOver`, since that is the callback function you want to call everytime the `mouseover` event triggers